File: solutions/3206-find-common-elements-between-two-arrays/solution.py
class Solution(object):
    def findIntersectionValues(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: List[int]
        """
        c,d=0,0
        a=set(nums1)
        b=set(nums2)
        for i in nums1:
            if i in b:
                c+=1
        for j in nums2:
            if j in a:
                d+=1
        return [c,d]

File: solutions/3206-find-common-elements-between-two-arrays/solution.java
class Solution {
    public int[] findIntersectionValues(int[] nums1, int[] nums2) {
        Set<Integer>ls1=new HashSet<>();
        Set<Integer>ls2=new HashSet<>();
        for (int num : nums1) ls1.add(num);
        for (int num : nums2) ls2.add(num);
        int c=0,d=0;
        for(int a:nums1)
            if(ls2.contains(a))
                c++;
        for(int a:nums2)
            if(ls1.contains(a))
                d++;
        return new int[]{c,d};
    }
}
